arbitrum-docs/run-arbitrum-node/nitro/01-build-nitro-locally.md

```diff
@@ -109,15 +109,15 @@ sudo mkdir -p /usr/local/bin
 sudo ln -s  /opt/homebrew/opt/llvm/bin/wasm-ld /usr/local/bin/wasm-ld
 ```
 
-### Step 5. Configure node [16.19](https://github.com/nvm-sh/nvm)
+### Step 5. Configure node [18.20.3](https://github.com/nvm-sh/nvm)
 
 #### For Debian/Ubuntu
 
 ```bash
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
 source "$HOME/.bashrc"
-nvm install 16.19
-nvm use 16.19
+nvm install 18.20.3
```

Comment:
Do we need to pin to the specific 18.20.3 version, or would it be sufficient to use "18" here?

Reply:
Good point, updated, thanks !